### PR TITLE
Feature/project page

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: src/components/project/project.component.jsx [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+              alt='Miroslav Milenkovic'
+            />
+            <span className={classes.name}>Miroslav Milenkovic</span>
+            <span>Methodology</span>
+          </div>
+          <div className={classes.profile}>
+            <img
+              className={classes.image}
+              src='./profiles/laia.jpg'
+              alt='Laia Soled'
```
Comment (AlbertoElias) [inline src/components/project/project.component.jsx:178]: Soler?

File: src/components/project/project.styles.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { createUseStyles } from 'react-jss'
+
+const useProjectStyles = createUseStyles(theme => ({
+  section: {
+    padding: theme.spacing(2),
+    [theme.breakpoints.sm]: {
+      padding: '96px 72px'
```
Comment (AlbertoElias) [inline src/components/project/project.styles.js:7]: theme.spacing?

File: src/components/project/project.styles.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(5)
+  },
+  full: {
+    width: '100%',
+    marginTop: theme.spacing(10),
+    marginBottom: theme.spacing(8)
+  },
+  profile: {
+    width: '214px'
```
Comment (AlbertoElias) [inline src/components/project/project.styles.js:60]: theme.spacing?

File: src/components/project/project.styles.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  image: {
+    width: '100%'
+  },
+  name: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+    color: theme.palette.black,
+    fontSize: '23px',
+    fontWeight: 500,
```
Comment (AlbertoElias) [inline src/components/project/project.styles.js:70]: theme.fontWeight?

**Review by AlbertoElias — APPROVED** [review]

File: src/components/shared/title/title.component.jsx [hunk 5]
```diff
@@ -35,7 +35,7 @@ function Title ({
       )
     case 'h4':
       return (
-        <h4 className={classes.title}>
+        <h4 className={`${classes.title} ${classes.titleSecondary}`}>
```
Comment (elias-garcia) [inline src/components/shared/title/title.component.jsx:38]: An `h4` heading will always be a `secondary` title? Or can we have an `h2` secondary title for example?

**Review by elias-garcia — COMMENTED** [review]
Just a question regarding the title component

File: src/components/project/project.component.jsx (hunk 1 above)
Comment (mmilenkovic) [reply src/components/project/project.component.jsx:178]: Oups 🤭

File: src/components/shared/title/title.component.jsx (hunk 5 above)
Comment (mmilenkovic) [reply src/components/shared/title/title.component.jsx:38]: Chami did not specify, so I added some headings for semantics and a11y. I would leave it like this and if we have any other needs (which I doubt) we can refactor this.

**Pull request merged**